app/articles/server/api/api.js

```diff
@@ -0,0 +1,104 @@
+import { Restivus } from 'meteor/nimble:restivus';
+import _ from 'underscore';
+
+import { processWebhookMessage } from '../../../lib';
+import { API } from '../../../api';
+import { settings } from '../../../settings';
+import * as Models from '../../../models';
+
+const Api = new Restivus({
+	enableCors: true,
+	apiPath: 'ghooks/',
+	auth: {
+		user() {
```

Comment:
Add rate-limiting, to prevent brute force

Reply:
Integrations also don't have any rate-limit, should I add it here. :thinking: